Find a functionally equivalent code:android.provider.DocumentsContract.createDocument:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * {@hide}
 */

@@ -1,8 +1,8 @@
 {
     final Bundle in = new Bundle();
-    in.putString(Document.COLUMN_DOCUMENT_ID, getDocumentId(parentDocumentUri));
+    in.putParcelable(DocumentsContract.EXTRA_URI, parentDocumentUri);
     in.putString(Document.COLUMN_MIME_TYPE, mimeType);
     in.putString(Document.COLUMN_DISPLAY_NAME, displayName);
     final Bundle out = client.call(METHOD_CREATE_DOCUMENT, null, in);
-    return buildDocumentUri(parentDocumentUri.getAuthority(), out.getString(Document.COLUMN_DOCUMENT_ID));
+    return out.getParcelable(DocumentsContract.EXTRA_URI);
 }

